src/System.Web.OData/OData/Query/Expressions/PropertyContainer.generated.tt

```diff
@@ -0,0 +1,64 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
```

Comment:
Do we really need the tt file to generate the codes?

Reply:
It's 32*4 classes.